lib/cucumber/rails/hooks/database_cleaner.rb

```diff
@@ -1,15 +1,21 @@
 # frozen_string_literal: true
 
 begin
-  require 'database_cleaner'
+  require 'database_cleaner/core'
+rescue LoadError
+  begin
+    require 'database_cleaner'
+  rescue LoadError
+    Cucumber.logger.debug('database_cleaner not present.')
```

Comment:
Probably worth hinting that this has checked both gems right?

Reply:
Sure! Are you imagining something like "neither database_cleaner v1 or v2 present"?

Reply:
Something like that yeh